generalTable.Rmd

```diff
@@ -85,25 +92,28 @@ trimmedData %<>%
 
 ```{r,echo=FALSE,results='hide',message=FALSE}
 # Create a General Table
-ICTR_general <- trimmedData %>%
+ITCR_general <- trimmedData %>%
   DT::datatable(
     style = 'default',
     width="100%",
-    colnames = c('Unique Factor' = 4, 'Types of Data' = 5,
-                 'Summary' = 3),
+    colnames = c('Unique Factor' = 4, 'Types of Data' = 5, 'Summary' = 3),
     rownames = FALSE,
     escape = FALSE,
+    #caption = "some caption",
```

Comment:
This might be where you could put the table keys and make it frozen - the caption.

Reply:
I tried freezing the table header, but that hasn't worked for some reason. If we make the table scrollable about Y-axis, it might be possible (haven't checked that yet) but I do not prefer that as you need scroll table and the page separately. Can we use different icons that are better representative of the object instead? Or, would you still prefer scrollable table?

Reply:
Yes we can use different icons- or go back to including the words

Reply:
Alright. Thank you! Could you tell me again where we got the original icons that we are using?

Reply:
looks like he used a png so you could use any free icon source to do that ... like iconfinder or iconpacks etc.

Reply:
The free icon websites ask that we attribute them (or the creator) for using the free icons. How do we do that?